splunksecrets.py

```diff
@@ -63,7 +63,8 @@ def decrypt(secret, ciphertext, nosalt=False):
         plaintext = "".join([six.unichr(c) for c in chars])
     elif ciphertext.startswith("$7$"):
         if len(secret) < 254:
-            raise ValueError(f"secret too short, need 254 bytes, got {len(secret)}")
+            print(f"secret too short ({len(secret)} bytes), padding to 254 bytes with nulls")
```

Comment:
Might be worth removing this since [the truncation of longer keys is already silent](https://github.com/HurricaneLabs/splunksecrets/blob/ae2f6282488c6f66d9027fbd8e84c1b7b6008920/splunksecrets.py#L132) but I could go either way.

If removed, the `if len(secret) < 254:` can go away entirely. :rocket:

Reply:
Thanks for the PR! I think it would make sense to just remove the message entirely personally.

Reply:
Removed in 1a10366.